src/main/java/org/javawebstack/orm/query/QueryCondition.java

```diff
@@ -1,23 +1,41 @@
 package org.javawebstack.orm.query;
 
-import org.javawebstack.orm.TableInfo;
+import org.javawebstack.orm.exception.ORMQueryException;
 
-import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
-import java.util.stream.Collectors;
-import java.util.stream.IntStream;
+import java.util.Locale;
 
 public class QueryCondition implements QueryElement {
 
+    private static final List<String> VALID_OPERATORS = Arrays.asList(
```

Comment:
Can we move this to a centralized information class? My suggestion would be something like `src/main/java/org/javawebstack/orm/information/QuerySyntax.java`.

I can reuse this information in tests and I can imagine it is reusable somewhere else.

I did something similiar here: https://github.com/JavaWebStack/ORM/pull/7/commits/cc4c851ed9b33fd85101fdfb3ae96f7ec3090811 (unmerged branch)

Reply:
In general I would consider static references to other classes bad practice and many other developers would probably agree. It creates dependencies and therefore often makes refactoring way harder. If we move it out of the class then i would suggest to create an enum for it which i probably should have done in the first place. This way we could even allow the enum as a parameter to the where method.

Reply:
Your right about that. Enums do work a lot better for this. I think there will be few cases where it's not the case.
Either way, I will keep it in mind for my future code.